packages/bot-web-ui/src/app/app.jsx

```diff
@@ -10,12 +9,13 @@ import {
     NetworkToastPopup,
     RoutePromptDialog,
 } from 'Components';
-import BlocklyLoading from '../components/blockly-loading';
-import { MobxContentProvider } from 'Stores/connect';
-import RootStore from 'Stores';
-import GTM from 'Utils/gtm';
 import BotBuilder from 'Components/dashboard/bot-builder';
+import GTM from 'Utils/gtm';
+import RootStore from 'Stores';
+import { MobxContentProvider } from 'Stores/connect';
+import BlocklyLoading from '../components/blockly-loading';
 import './app.scss';
+import '../public-path'; // Leave this here (at the top)! OK boss!
```

Comment:
@maryia-matskevich-deriv can we remove this import? Is it something we are using?

Reply:
why it should be here even?
i mean `Leave this here` is sort of a command to leave that import there without changing?

Reply:
moved to the first line & added the rule in the eslint config